hooks/pre-commit

```diff
@@ -23,7 +23,7 @@ chronic typos .
 (cd relayer && chronic mage lint && chronic go fmt ./...)
 
 # format rust
-(cd parachain && chronic cargo +nightly fmt)
+(cd parachain && chronic cargo +nightly-"$SNOWBRIDGE_RUST_NIGHTLY" fmt)
```

Comment:
Nice, I'll add that to https://github.com/Snowfork/snowbridge/pull/920

Reply:
I'll remove the formatting here & leave this change to #920